python/ray/test_utils.py

```diff
@@ -542,3 +542,29 @@ def load_test_config(config_file_name):
 def set_setup_func():
     import ray._private.runtime_env as runtime_env
     runtime_env.VAR = "hello world"
+
+
+def get_nightly_url() -> str:
```

Comment:
don't think this is used anymore

Reply:
Do you think it's okay to still merge it?  I made an issue to remove this and some other things that are now unused https://github.com/ray-project/ray/issues/15912